pcsx2-xp.txt

```diff
@@ -0,0 +1,15 @@
+[Section]
+Name = PCSX2 (XP Compatible)
+Version = 1.6.0
+License = GNU GPL v2
+Description = PCSX2 is a Playstation 2 emulator. It needs Microsoft Visual C++ 2013 redistributable.
+Category = 4
+URLSite = http://neonfloppy.sytes.net/projects/pcsx2-xp/
+URLDownload = http://neonfloppy.sytes.net/projects/pcsx2-xp/releases/1.6.0/pcsx2-xp-binaries.7z
```

Comment:
I double-checked already: http**s** doesn't work for those 2 links on Windows.

Reply:
So I suggest two options:
1. Trust the http anyway until we have support for Vista+ functions and then update the links to the common normalized version.
2. Change the links through Webarchive, that drops the speed but is more trusty.

Reply:
No change requested necessarily. I just wrote that, to prevent others recommending to change the URL to http**s** (without them testing the result).
Our additional hash should guarantee integrity also when transferring via http.